### ISSUE TITLE
Long path to reach a specific tutorial from the website

### ISSUE THREAD

**Issue by vepadulano** [root-project/web#61]
If a user is searching for a specific feature tutorial, starting from the front page the path is:
Get Started --> Code Examples --> "These" link in the middle of the page text --> Click on the tutorial topic --> Get to the final tutorial.

So I believe that the first click is ok, but then I would like the "Code examples" link to bring me directly to the tutorial list. I would also agree with #17 that there should be a consistent naming for the tutorials.

If we want to differentiate between standard tutorials and links to SWAN it's fine, but I would make it more evident where one should click than using "These" link in the middle of the text

**Comment by Axel-Naumann** [comment]
Can we move the content of https://root-project.github.io/web/get_started/code_examples/ into https://root.cern.ch/doc/master/group__Tutorials.html, and have the "Code Examples" link go directly to https://root.cern.ch/doc/master/group__Tutorials.html ?

**Comment by Axel-Naumann** [comment]
Relevant for the publication because it will change the URL (if we get rid of the "Code Examples" page)

**Comment by eguiraud** [comment]
Related issues if you want to shuffle this content around: #70 #17

**Comment by hageboeck** [comment]
Shouldn't we have `Tutorials` directly on the front page?

**Comment by couet** [comment]
Indeed, from the front page, it is only two clicks Reference->Tutorials.
